python/ray/autoscaler/_private/_azure/node_provider.py

```diff
@@ -265,14 +332,17 @@ def _create_node(self, node_config, tags, count):
         config_tags.update(tags)
         config_tags[TAG_RAY_CLUSTER_NAME] = self.cluster_name
 
-        vm_name = "{node}-{unique_id}-{vm_id}".format(
+        # Use a unique name for the deployment
+        deployment_name = "{node}-{unique_id}-{vm_id}".format(
             node=config_tags.get(TAG_RAY_NODE_NAME, "node"),
             unique_id=self.provider_config["unique_id"],
             vm_id=uuid4().hex[:UNIQUE_ID_LEN],
         )[:VM_NAME_MAX_LEN]
 
         template_params = node_config["azure_arm_parameters"].copy()
-        template_params["vmName"] = vm_name
+        # The vmName parameter in the template is now a prefix.
```

Comment:
I don't understand these comments, I don't see any changes to the vmName parameter

Reply:
whoops, i changed the behavior in a previous iteration and the comments are no longer applicable.
Good catch, let me update